app/src/main/java/com/kickstarter/ui/viewholders/FrequentlyAskedQuestionsViewHolder.kt

```diff
@@ -1,35 +1,43 @@
 package com.kickstarter.ui.viewholders
 
 import com.kickstarter.databinding.ItemFrequentlyAskedQuestionCardBinding
-import com.kickstarter.libs.rx.transformers.Transformers
+import com.kickstarter.libs.utils.extensions.addToDisposable
 import com.kickstarter.models.ProjectFaq
 import com.kickstarter.viewmodels.FrequentlyAskedQuestionsViewHolderViewModel
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
 
 class FrequentlyAskedQuestionsViewHolder(
     val binding: ItemFrequentlyAskedQuestionCardBinding
 ) : KSViewHolder(binding.root) {
 
-    private val viewModel: FrequentlyAskedQuestionsViewHolderViewModel.ViewModel =
-        FrequentlyAskedQuestionsViewHolderViewModel.ViewModel(environment())
+    private val viewModel: FrequentlyAskedQuestionsViewHolderViewModel.FrequentlyAskedQuestionsViewHolderViewModel =
+        FrequentlyAskedQuestionsViewHolderViewModel.FrequentlyAskedQuestionsViewHolderViewModel()
+    private val disposables = CompositeDisposable()
 
     init {
         viewModel.outputs.question()
-            .compose(bindToLifecycle())
-            .compose(Transformers.observeForUI())
+            .observeOn(AndroidSchedulers.mainThread())
             .subscribe { binding.questionAnswerLayout.setQuestion(it) }
+            .addToDisposable(disposables)
 
         viewModel.outputs.answer()
-            .compose(bindToLifecycle())
-            .compose(Transformers.observeForUI())
+            .observeOn(AndroidSchedulers.mainThread())
             .subscribe { binding.questionAnswerLayout.setAnswer(it) }
+            .addToDisposable(disposables)
 
         viewModel.outputs.updatedDate()
-            .compose(bindToLifecycle())
-            .compose(Transformers.observeForUI())
+            .observeOn(AndroidSchedulers.mainThread())
             .subscribe { binding.questionAnswerLayout.setLastedUpdateDate(it) }
+            .addToDisposable(disposables)
     }
 
     override fun bindData(data: Any?) {
         this.viewModel.inputs.configureWith(data as ProjectFaq)
     }
+
+    override fun destroy() {
```

Comment:
When it's a ViewHolder, you need to make sure the Adapter is calling destroy to clean the disposables, In this case `FrequentlyAskedQuestionsAdapter` inherits from `KSListAdapter` you need to add on the `onViewDetachedFromWindow
<img width="636" alt="Screenshot 2023-06-29 at 2 02 50 PM" src="https://github.com/kickstarter/android-oss/assets/4083656/d4d0477e-aa70-4c30-befe-71620701fb28">
` the active call to `destroy`,  that before was handled on `ActivityViewModel`
<img width="636" alt="Screenshot 2023-06-29 at 2 02 50 PM" src="https://github.com/kickstarter/android-oss/assets/4083656/81a9d076-580f-4e2a-891f-689d9737db74">

Reply:
got it, updating now